docs/CONTAINER_INTERFACE.md

```diff
@@ -121,6 +121,13 @@ manager, please consider supporting the following interfaces.
    `container_ttys=pts/7 pts/8 pts/14` it will spawn three additional login
    gettys on ptys 7, 8, and 14.
 
+4. To allow applications to detect the OS version and other metadata of the host
+   running the container manager, please parse the host's `/etc/os-release` and
+   for each key set a `$<TYPE>_host_KEY=VALUE` environment variable, with `<TYPE>`
+   being `service` when the guest only wraps a single service (eg: `portable services`,
+   `nspawn` with a stub init) and `container` when wrapping a complete image
+   (eg: `nspawn` with a fully-fledged init).
```

Comment:
`CONTAINER_INTERFACE.md` is focussed on systemd as the payload, and it's written from the perspective of pid1 int he container being systemd. i.e. it documents what systemd in the container expects from the enveloping container manager. I don't think we should mention the portable service stuff here hence. i.e. be explicit, just say `$container_host_id=`, `$container_host_version_id=` and so on. 

Also, I think we should only import the xyz_ID fields from os-release, not the others.